tests/exclude_system_edl/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Open Enclave SDK contributors.
```

Comment:
Could you add a README.md or some description as to what this test is supposed to illustrate? Is this just a temporary test that will be removed once all tests are updated to import system EDLs explicitly?

Reply:
This is just a temporary test to make sure everything works when COMPILE_SYSTEM_EDL=OFF. I can add a README.md to explain that

Reply:
Added a README